tests/build-conf/beagleboneblack/bblayers.conf

```diff
@@ -10,5 +10,6 @@ BBLAYERS ?= " \
   @WORKSPACE@/meta-poky \
   @WORKSPACE@/meta-yocto-bsp \
   @WORKSPACE@/meta-mender/meta-mender-core \
+  @WORKSPACE@/meta-openembedded/meta-oe \
```

Comment:
Actually, for `beagleboneblack` we don't want to add this layer, as we will not test it. The rest are fine.